src/main/java/com/berrie/gamerental/controller/ExceptionHandlerController.java

```diff
@@ -35,6 +35,6 @@ public ErrorResponse handleUserExistsException(UserExistsException ex) {
 
     @ExceptionHandler(UserNotFoundException.class)
```

Comment:
consider making this exception UserUnauthorizedException, it's more fitting to the use cases